api/controllers/zetter.js

```diff
@@ -14,9 +15,22 @@ const getTweets = (req, res) => {
         }
     });
 
+    // ツイートに自分がfavしているかの情報を付加するための準備
+    const usersFavoriteTweets = favorities.filter((favorite) => favorite.userId === req.user.id);
+    const favoriteTweetIds = usersFavoriteTweets.map((obj) => obj.tweetId);
+
     tweets.forEach((tweet) => {
-        const user = users.find((user) => user.id === tweet.ownerId);
+        const user = users.find(({ id }) => id === tweet.ownerId);
         tweet['user'] = user;
+
+        const numberOfFavorite = favorities.filter((favorite) => favorite.tweetId === tweet.id);
+        tweet['numberOfFavorite'] = numberOfFavorite.length;
```

Comment:
```suggestion
        const favoritiesOfTweet = favorities.filter((favorite) => favorite.tweetId === tweet.id);
        tweet['numberOfFavorite'] = favoritiesOfTweet.length;
```
`numberOfFavorite`自体は「数」ではないので、変数名は別のにしておいてもいいかも

たんに`tweetFavorities`とかのほうがわかりやすいかもしれないけど